### ISSUE TITLE
Scripting: "PlayLooped:" command not recognised

### ISSUE THREAD

**Issue by SteveDaulton** [audacity/audacity#2266]
**Describe the bug**
Audacity 3.2.0 alpha does not recognise the command "PlayLooped:"

**To Reproduce**
1. Ensure that mod-script-pipe is enabled
2. Generate a sound and ensure that it is selected
3. From Python, send the "PlayLooped" command (as listed in the [command reference](https://manual.audacityteam.org/man/scripting_reference.html))
4. Observe Error:
```
Your batch command of PlayLooped was not recognized.
BatchCommand finished: Failed!
```

**Expected behavior**
The selected audio to loop play

**Screenshots**

**Additional information (please complete the following information):**
 - OS: Xubuntu
 - Version 3.2.0 alpha
 - (if relevant) Sound Device [e.g. Windows WASAPI, USB Sound card]

**Additional context**
Maybe better to replace the "PlayLooped" command with a new "Loop:" command that has a parameter: 
"*bool **Enabled** (default False)*"

**Comment by LWinterberg** [comment]
actually, is this a bug with Audacity or a bug with the scripting reference?

**Comment by SteveDaulton** [comment]
> actually, is this a bug with Audacity or a bug with the scripting reference?

As I see it, there's two issues:

1. There is no longer any way to loop play via macro / scripting commands.
2. Documentation: The scripting reference is wrong.

Unfortunately, closing bugs without a fix does not make the issues go away for users, though it could make users go away from using the app.

**Comment by LWinterberg** [comment]
So: 

1. would be fixed by https://github.com/audacity/audacity/issues/3326
2. would be fixed by removing it from the list. I can do that.

**Comment by SteveDaulton** [comment]
Yes I think so, though there also appears to be some new commands that are not yet documented. The ones that I've spotted are:

* Set Loop In = SetPlayRegionIn:
* Set Loop Out = SetPlayRegionOut:
* Set Loop to Selection = SetPlayRegionToSelection
* Loop On/Off = TogglePlayRegion

I don't know if there are any others.

**Comment by tin2tin** [comment]
(The missing PlayLooped: is breaking this quite popular add-on which allows users of Blender VSE to edit their audio in Audacity): https://github.com/tin2tin/audacity_tools_for_blender

**Comment by tin2tin** [comment]
Users are complaining over the Blender add-on for round-tripping audio in Audacity is broken due to the missing "PlayLooped:" command in Audacity. So, what is the status on this? Or are there any workarounds I can use instead?
